packages/charting/src/components/StackedBarChart/StackedBarChart.styles.ts

```diff
@@ -45,6 +45,16 @@ export const getStyles = (props: IStackedBarChartStyleProps): IStackedBarChartSt
     },
     opacityChangeOnHover: {
       opacity: shouldHighlight ? '' : '0.1'
+    },
+    ratioNumerator: {
+      fontSize: '12px',
+      fontWeight: '600',
```

Comment:
use import { DefaultFontStyles, DefaultPalette, FontSizes, FontWeights, IStyle } from 'office-ui-fabric-react/lib/Styling';

Reply:
DefaultFontStyles and DefaultPalette shouldn't be used here.